spec/factories/addresses.rb

```diff
@@ -0,0 +1,4 @@
+FactoryBot.define do
+  factory :address do
+  end
```

Comment:
```suggestion
  factory :address do
    user
    address { FFaker::AddressUA.street_address }
    city { FFaker::AddressUA.city }
    zip { FFaker::AddressUA.zip_code }
    country { FFaker::AddressUA.country }
    phone { FFaker::PhoneNumberUA.home_phone_number }
  end
  
  factory :shipping, parent: :address, class: 'Shipping' {}
  factory :billing, parent: :address, class: 'Billing' {}
```
And remove `spec/factories/billings.rb` and `spec/factories/shippings.rb` files.